src/app/test_executive/test_executive.ml

```diff
@@ -48,6 +48,8 @@ let tests : test list =
   ; ("delegation", (module Delegation_test.Make : Intf.Test.Functor_intf))
   ; ("archive-node", (module Archive_node_test.Make : Intf.Test.Functor_intf))
   ; ("gossip-consis", (module Gossip_consistency.Make : Intf.Test.Functor_intf))
+  ; ( "opt-block-prod"
```

Comment:
see earlier comment about name length

Reply:
Yeah, I know the issue, but as evidenced by green CI, I'm not surpassing the limit